it/controllers/correctReturn/PackAtBusinessAddressControllerISpec.scala

```diff
@@ -237,15 +241,17 @@ class PackAtBusinessAddressControllerISpec extends ControllerITTestHelper {
                 client, correctReturnBaseUrl + checkRoutePath, Json.obj("value" -> yesSelected.toString)
               )
 
-              whenReady(result) { res =>
-                res.status mustBe 303
-                res.header(HeaderNames.LOCATION) mustBe Some(routes.CorrectReturnCYAController.onPageLoad.url)
-                val updatedAnswers = getAnswers(userAnswers.id)
-                val dataStoredForPage = updatedAnswers.fold[Option[Boolean]](None)(_.get(PackAtBusinessAddressPage))
-                dataStoredForPage.nonEmpty mustBe true
-                dataStoredForPage.get mustBe yesSelected
-                updatedAnswers.map(data => data.packagingSiteList) mustBe
-                  Some(if (yesSelected) packagingSitesFromSubscription ++ packAtBusinessAddressSite else packagingSitesFromSubscription)
+              if(yesSelected){
```

Comment:
Need an equivalent test for when no is selected - currently it's not being tested